### PR TITLE
Fix typo on threshold

### PR DESCRIPTION
**Pull request by germanurrus**
Fix typo and validation

### CODE REVIEW

File: frontend/packages/client/src/components/Community/ProposalThresholdEditor/FormConfig.js [hunk 1]
```diff
@@ -30,7 +30,11 @@ const Schema = (isValidFlowAddress) =>
       .when('onlyAuthorsToSubmitProposals', {
         is: true,
         then: addresValidation(yup.string(), isValidFlowAddress, true),
-        otherwise: addresValidation(yup.string(), isValidFlowAddress, false),
+        otherwise: addresValidation(
+          yup.string().required('Please enter a contract address'),
+          isValidFlowAddress,
+          false
+        ),
```
Comment (germanurrus) [inline frontend/packages/client/src/components/Community/ProposalThresholdEditor/FormConfig.js:37]: when all fields are empty this makes the required go first, then all error messages are about fields being required
Screen Shot 2022-08-16 at 17 03 37

File: frontend/packages/client/src/components/Community/ProposalThresholdEditor/FormConfig.js [hunk 2]
```diff
@@ -30,7 +30,11 @@ const Schema = (isValidFlowAddress) =>
       .when('onlyAuthorsToSubmitProposals', {
         is: true,
         then: addresValidation(yup.string(), isValidFlowAddress, true),
-        otherwise: addresValidation(yup.string(), isValidFlowAddress, false),
+        otherwise: addresValidation(
```
Comment (jbluks) [inline frontend/packages/client/src/components/Community/ProposalThresholdEditor/FormConfig.js:33]: it looks like this is missing an s - addres**s**Validation

**Review by jbluks — APPROVED** [review]
One small typo.

File: frontend/packages/client/src/components/Community/ProposalThresholdEditor/FormConfig.js (hunk 2 above)
Comment (germanurrus) [reply frontend/packages/client/src/components/Community/ProposalThresholdEditor/FormConfig.js:33]: thanks Jonathan! getting it fixed!

**Pull request merged**